Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -942,37 +952,45 @@ def detonate_sample_output(response_json, sha1):
     return results
 
 
-def dynamic_analysis_results_command():
-    sandbox = DynamicAnalysis(
-        host=TICLOUD_URL,
-        username=USERNAME,
-        password=PASSWORD,
-        user_agent=USER_AGENT,
-        proxies=PROXIES,
-        verify=VERIFY_CERTS
-    )
+def sample_dynamic_analysis_results_command():
+    da = create_da_object()
 
     sha1 = demisto.getArg("sha1")
+    analysis_id = demisto.getArg("analysis_id")
+    latest_analysis = argToBoolean(demisto.getArg("latest_analysis"))
 
     try:
-        response = sandbox.get_dynamic_analysis_results(sample_hash=sha1, latest=True)
+        response = da.get_dynamic_analysis_results(
+            sample_hash=sha1,
+            analysis_id=analysis_id if analysis_id else None,
+            latest=latest_analysis if latest_analysis else None
+        )
     except Exception as e:
+        if hasattr(e, "response_object"):
+            return_error(f"status code: {e.response_object.status_code}, message: {e.response_object.text}")
+
         return_error(str(e))
 
     response_json = response.json()
-
-    results, file_results = dynamic_analysis_results_output(response_json, sha1)
+    results, file_results = sample_dynamic_analysis_results_output(response_json, sha1)
 
     return_results([results, file_results])
 
 
-def dynamic_analysis_results_output(response_json, sha1):
+def sample_dynamic_analysis_results_output(response_json, sha1):
     classification = response_json.get("rl", {}).get("report", {}).get("classification")
-    classification = classification.upper()
     md5 = response_json.get("rl", {}).get("report", {}).get("md5")
     sha256 = response_json.get("rl", {}).get("report", {}).get("sha256")
+    last_analysis = response_json.get("rl", {}).get("report", {}).get("last_analysis")
 
-    d_bot_score = classification_to_score(classification)
+    markdown = f"""## ReversingLabs Sample Dynamic Analysis output for sample {sha1}\n **Classification**: {classification}
+    **Sample SHA1**: {sha1}
+    **Sample MD5**: {md5}
+    **Sample SHA256**: {sha256}
+    **Last analysis**: {last_analysis}\n ### Full report is returned as JSON in a downloadable file
+    """
```

Comment:
You can use the `tableToMarkdown` method which is implemented in the `commonserverpython.py` file. Not mandatory, but much easier to use :)

Reply:
I know, thank you. I use that method on some outputs but in this case I find it better to manually write these couple of lines.